### DISCUSSION TITLE
Stage selection

### DISCUSSION THREAD

**Discussion by joakim-hove** [NeogitOrg/neogit#1565] [Q&A] [answered]
Hello;

in magit - which is where I come from - it was possible to commit a selection. I.e. if i have the following diff:
```
+ new_line1
+ new_line2
+ new_line3
```
and I only want to stage and commit the `new_line2`. Is that possible in neogit? If I try to press 'v' to start a selection I get a menu with some seemingly unrelated changes.

**Comment by CKolkey** [comment, marked as answer]
Use linewise visual selection - `V` :)

**Reply by joakim-hove** [reply]
That works - cool ;-)
